desktop.css

```diff
@@ -0,0 +1,206 @@
+@media (min-width: 768px) {
+  body {
+    width: 1440px;
+    background: #f7f7f9;
+  }
+
+  .about,
+  .work,
+  .headline,
+  .contact {
+    margin: auto;
+    width: 100%;
+  }
+
+  header {
+    width: 100%;
+    padding: 26px 154px;
+    position: sticky;
+    top: 0;
+    margin: auto;
+  }
+
+  .menu_main .humbuger {
+    width: 0;
+    height: 0;
+  }
+
+  .menu_main .menu {
+    position: none;
+    visibility: visible;
+    flex-direction: row;
+    overflow: visible;
+    gap: 32px;
+  }
+
+  .menu li {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .headline {
+    top: 72px;
+    width: 100%;
+    left: 0%;
+    background-image: url(imgoverlay/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-position-x: 0;
+    background-position-y: 100%;
+  }
+
+  .container {
+    gap: 12px;
+    padding: 100px 260px;
+    width: auto;
+    height: auto;
+    overflow: none;
+  }
+
+  .headline h1 {
+    width: auto;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    height: auto;
+    padding-right: 500px;
+    padding-bottom: 12px;
+    overflow: none;
+  }
```

Comment:
- [ ] **[OPTIONAL]** On the desktop version there are two lines in the Figma version but in yours, there are three, please 🙏🏽 correct it to be similar to the Figma version.

| Figma version | Your version |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/206596880-e99ee51d-21ad-4e3f-95c0-d09e1fb9d648.png) | ![image](https://user-images.githubusercontent.com/58642949/206596830-9ea34c25-6f3d-4c14-8065-57672a0f0aae.png) |